modules/rosa-sdpolicy-account-management.adoc

```diff
@@ -99,10 +101,11 @@ The following AWS regions are supported by Red Hat OpenShift 4 and are supported
 - ca-central-1 (Central)
```

Comment:
This should probably be "Central Canada", I'm going to update the OSD Service Definition now

Reply:
You got it.